guis/qml/qml/MainQML.qml

```diff
@@ -0,0 +1,159 @@
+        root.handler = ContentHandler.Source
+        root.selectionType = ContentTransfer.Multiple
+      } else if(request.message =="photo"){
+        root.request = request
+        root.requestContentHub = true
+        root.contentType = ContentType.Pictures
+        root.handler = ContentHandler.Source
+        root.selectionType = ContentTransfer.Single
+      } else if(request.message =="video"){
+        root.request = request
+        root.requestContentHub = true
+        root.contentType = ContentType.Videos
+        root.handler = ContentHandler.Source
+        root.selectionType = ContentTransfer.Single
+      } else if(request.message =="document"){
+        root.request = request
+        root.requestContentHub = true
+        root.contentType = ContentType.Documents
+        root.handler = ContentHandler.Source
+        root.selectionType = ContentTransfer.Single
+      } else if(request.message =="audio"){
+        root.request = request
+        root.requestContentHub = true
+        root.contentType = ContentType.Music
+        root.handler = ContentHandler.Source
+        root.selectionType = ContentTransfer.Single
+      } else if(request.message.includes("[oC]")){
+        root.request = request
+        root.requestContentHub = true
+        root.url = request.message.substring(4)
+        root.contentType = ContentType.Contacts
+        root.handler = ContentHandler.Destination
+        root.selectionType = ContentTransfer.Multiple
+      } else if(request.message.includes("[oP]")){
+        root.request = request
+        root.requestContentHub = true
+        root.url = request.message.substring(4)
+        root.contentType = ContentType.Pictures
+        root.handler = ContentHandler.Destination
+        root.selectionType = ContentTransfer.Single
+      } else if(request.message.includes("[oV]")){
+        root.request = request
+        root.requestContentHub = true
+        root.url = request.message.substring(4)
+        root.contentType = ContentType.Videos
+        root.handler = ContentHandler.Destination
+        root.selectionType = ContentTransfer.Single
+      } else if(request.message.includes("[oD]")){
+        root.request = request
+        root.requestContentHub = true
+        root.url = request.message.substring(4)
+        root.contentType = ContentType.Documents
+        root.handler = ContentHandler.Destination
+        root.selectionType = ContentTransfer.Single
+      } else if(request.message.toLowerCase().includes("call")){
+        root.request = request
+        var callUrl = request.message.substring(4)
+        Qt.openUrlExternally("tel:///"+callUrl);
+        request.dialogAccept();
+
+      } else if(request.message.toLowerCase().includes("http")){
+          Qt.openUrlExternally(request.message);
+          request.dialogAccept();
+      } else{
+      simpleDialog.request = request;
+      simpleDialog.visible = true;
+      root.request = request;
```

Comment:
this all can be removed, too because it'°s use by ubuntu touch for the file dialog. Îf you can send attachments on debian then you don't need it anymore.